style-guide/src/pages/404.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from '../styles/index.module.scss';
+import SectionHeader from '../components/section-header';
+import withLayout from '../hocs/withLayout';
+
+export default withLayout(() => {
+  return (
+    <div className={styles.mainContainer}>
+      <SectionHeader text="Whoops" />
+      <p>404</p>
+    </div>
```

Comment:
Running locally 404 works, I don't think it will once we've built and deployed to gh-pages